test-runner/test_runner.py

```diff
@@ -205,7 +205,8 @@ def get_test_list(args: dict, tests_yaml: List[dict]):
         remaining_containers = docker.container.list()
         for container in remaining_containers:
             docker.stop(container, time=None)
-        docker.image.remove(test_images, force=True)
+        docker.image.remove(test_images, force=True, prune=False)
```

Comment:
Any particular reason why we need a separate prune and can't use the prune param in this function?

Reply:
It leaves behind some images, and causes our GPU runners to run out of space after multiple preset `test-containers` jobs.

Reply:
This is a regression in `python_on_whales`. I validated that issue locally and say that by running system prune it doesn't leave the same images behind.